starlite/plugins/sql_alchemy.py

```diff
@@ -284,15 +284,19 @@ def to_pydantic_model_class(self, model_class: DeclarativeMeta, **kwargs: Any) -
                 # to avoid duplication of pydantic models, we are using forward refs
                 # see: https://pydantic-docs.helpmanual.io/usage/postponed_annotations/
                 for name, relationship_property in mapper.relationships.items():
-                    related_entity_class = relationship_property.mapper.class_
-                    related_model_name = related_entity_class.__qualname__
-                    if relationship_property.uselist:
-                        field_definitions[name] = (List[related_model_name], ...)  # type: ignore
+                    if not relationship_property.back_populates and not relationship_property.backref:
+                        related_entity_class = relationship_property.mapper.class_
+                        related_model_name = related_entity_class.__qualname__
+                        if relationship_property.uselist:
+                            field_definitions[name] = (Optional[List[related_model_name]], None)  # type: ignore
```

Comment:
Wouldn't this just simplify into `field_definitions[name] = Optional[List[related_model_name]]` ?

Reply:
in this case though pydantic will fail if None value is given.